iodata/test/test_wfn.py

```diff
@@ -22,20 +22,24 @@
 
 
 import numpy as np
-from .common import compute_mulliken_charges, get_fn, check_normalization
+from .common import compute_mulliken_charges, check_normalization
 from ..wfn import load_wfn_low, get_permutation_basis, get_permutation_orbital, get_mask
 from ..iodata import IOData
 from ..overlap import compute_overlap
 from ..utils import shells_to_nbasis
 
+try:
+    from importlib_resources import path
+except ImportError:
+    from importlib.resources import path
 
 # TODO: removed density, kin, nucnuc checks
 
 
 def test_load_wfn_low_he_s():
-    fn_wfn = get_fn('he_s_orbital.wfn')
-    title, numbers, coordinates, centers, type_assignment, exponents, \
-    mo_count, occ_num, mo_energy, coefficients, energy = load_wfn_low(fn_wfn)
+    with path('iodata.test.cached', 'he_s_orbital.wfn') as fn_wfn:
+        title, numbers, coordinates, centers, type_assignment, exponents, \
+        mo_count, occ_num, mo_energy, coefficients, energy = load_wfn_low(str(fn_wfn))
```

Comment:
This line should be indented by four spaces, I believe.
```suggestion
            mo_count, occ_num, mo_energy, coefficients, energy = load_wfn_low(str(fn_wfn))
```